SonarQube.Client/SonarQubeService.cs

```diff
@@ -47,14 +47,35 @@ public class SonarQubeService : ISonarQubeService, IDisposable
 
         public Version SonarQubeVersion { get; private set; }
 
-        public bool HasOrganizationsFeature
+        public async Task<bool> HasOrganizations(CancellationToken token)
```

Comment:
This is now a function call rather than a property because it does more than just a simple numerical check.